README.md

```diff
@@ -1,3 +1,7 @@
+![Standalone](https://github.com/kokkos/kokkos-tools/actions/workflows/build-standalone.yml/badge.svg)
+![Kokkos@develop](https://github.com/kokkos/kokkos-tools/actions/workflows/build-with-kokkos.yml/badge.svg)
+![clang-format](https://github.com/kokkos/kokkos-tools/actions/workflows/clang-format-check.yml/badge.svg)
+
```

Comment:
No problem, I'll revert this ;) I was primarily just a mean to trigger the pipeline again (since https://github.com/kokkos/kokkos/pull/6114 was merged in the Kokkos develop branch, I wanted to check all would be fine).